recipe/meta.yaml

```diff
@@ -27,9 +25,10 @@ requirements:
 test:
   requires:
     - pkg-config
-  files:
-    - run_test.sh         # [osx]
-    - run_test.py         # [osx]
+  # files:
+    # - run_test.sh         # [osx]
```

Comment:
Are we able to run the shell tests or are those also not working for some reason?

Reply:
There were some problems in this build: https://travis-ci.org/conda-forge/ncurses-feedstock/builds/368212284

There was a weird infinite loop right after calling the shell script. I couldn't trace the origin of the problem (it could be unrelated), but commenting this out fixed it.

Reply:
Anyway, we should put the test to run again after Python is rebuild with this newest `ncurses` version